Contrast-Security-OSS/NodeTestBenches
# Node 873 stub dynamodb sinks

Steps to Verify:

0. `npm i && npm link` in `test-bench-utils/` and `npm link @contrast/test-bench-utils` in whatever test bench app you use.
1. Spin up any test bench app 
2. Hit the routes and confirm `nosql-injection` gets reported

![Screen Shot 2020-06-12 at 3 57 05 PM](https://user-images.githubusercontent.com/14120224/84541500-85b52f00-acc5-11ea-8cdf-3a1b081f3ee4.png)

Additional Notes: 
- Once you hit the route you might see the rule report twice. I spent some time trying to dig in to why this was happening but could not discover the root cause. It looks like the post-hook runs twice for some reason
